src/components/RadioButton/RadioButton.st.css

```diff
@@ -158,6 +158,7 @@
     width: 100%;
     font: font(value(LabelTextFont));
     color: applyOpacity(value(CurrentTextColor),1);
+    line-height: 16px;
```

Comment:
The line-height property is defined inside the 'LabelTextFont' var and set to 1.5em.
As I see in the spec in our ZeroHeights the line-height should be 24px which is 1.5em.

`LabelTextFont: "{theme: 'Body-M', size: '16px', lineHeight: '1.5em'}";`

![Screen Shot 2020-12-27 at 15 53 39](https://user-images.githubusercontent.com/66675260/103172378-b15fc700-485b-11eb-96d2-f177a495606b.png)

https://zeroheight.com/7sjjzhgo2/p/05a744-05-selections/b/285e96

Reply:
Why do you need to change it?
If there is a problem let's talk with @michalvardi and consider changing it :)

Reply:
The issue is that having the 24px line-height causes the total height to be 58px, while in zeroheight it's 50px. If that is not a problem feel free to close this.

Reply:
From what I can see in zeroheight, it should actually be `56px`.
So maybe instead of changing the lineheight, we need to change the padding?

Reply:
![image](https://user-images.githubusercontent.com/3277080/103174314-baf02b80-4869-11eb-9809-5e938a47a260.png)

Reply:
So it seems that there is no need to change anything...
There is padding of 16px top an bottom so it's 16+16+ 24(line height) = 56px

Reply:
You would think but it is actually 58px, I'm not sure why. I guess the border is adding to the height for some reason.

Reply:
I got it.. The border is 1px so yes we could reduce the padding